src/main/java/lab2/Lab2.java

```diff
@@ -1,60 +1,94 @@
+// it creates an array of strings / type: string
     // Pick a random style
     String[] styles = { "adventurer", "adventurer-neutral", "avataaars", "big-ears", "big-ears-neutral", "big-smile", "bottts", "croodles", "croodles-neutral", "fun-emoji", "icons", "identicon", "initials", "lorelei", "micah", "miniavs", "open-peeps", "personas", "pixel-art", "pixel-art-neutral" };
-    var style = styles[(int)(Math.random() * styles.length)];
+       var style = styles[(int)(Math.random() * styles.length)]; // picks a random index from the array
+    // math.random() is a class method from java.lang.Math that returns between 0 and 1
+    // styles.lenght = variable that returns the number of items in the array
+    // style is a string - ref type
 
     // Generate a random seed
-    var seed = (int)(Math.random() * 10000);
+    var seed = (int)(Math.random() * 10000); // creates a random number between 0 and 9999 / seed is an integer - primitive type
 
     // Create an HTTP request for a random avatar
+    // formats a URL string using the style and seed
+    // URI.create is a static class that creates URI object from a string
+    // URI is a class from java.net
     var uri = URI.create("https://api.dicebear.com/9.x/%s/png?seed=%d".formatted(style, seed));
-    var request = HttpRequest.newBuilder(uri).build();
+
+    // create an http request using the uri
+    var request = HttpRequest.newBuilder(uri).build(); // is a static class - returns a builder object / .build() instance that creates httprequest object / request is an httprequest for the reference type
 
     // Send the request
+    // creates a new http client
     try (var client = HttpClient.newHttpClient()) {
+        // httpresponse.bodyhandlers.ofinputstream() is a static method that gives instructions to return the body as inpustream
+        // response is an httpresponse object
         var response = client.send(request, HttpResponse.BodyHandlers.ofInputStream());
-        return response.body();
+        return response.body(); // ,body() is an instance that returns the body of the response as an inputstream
     }
 }
-
+// it takes an inputstream (ref type) and displays the image
 void showAvatar(InputStream imageStream) {
+    // create a new windows
+    // jframe is a class from javax.swing
+    // frame is a JFrame ref type
     JFrame frame = new JFrame("PNG Viewer");
+    // setdefaultcloseoperation is a method that tells the window to close when X is clicked
     frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+    // setresizable is a method when false means the user cant resize the window
     frame.setResizable(false);
+    // setsize sets the width and height of the window
     frame.setSize(200, 200);
+    // getcontentpane() returns a container
+    // .setbackground(color.black) sets the background color
+    // color.black is a variable from java.awt.color (class for color values)
     frame.getContentPane().setBackground(Color.BLACK);
```

Comment:
These are all instance methods.  There are also two class variales: EXIT_ON_CLOSE and BLACK